### ISSUE TITLE
app updater fails to upgrade app to 2020.3.0

### ISSUE THREAD

**Issue by julien-nc** [marius-wieschollek/passwords#248]
Hey there. In my setup, it's impossible to upgrade Passwords app to 2020.3.0 from 2020.2.1.

Nextcloud 18.0.1.3 with SQLite3.

There is a complain about adding a "NOT NULL" field with default value NULL (which is I think in fact a missing default value in `database.xml`). Here is the corresponding log from `nextcloud.log`:
```
"message":"Failed to update database structure (PDOException: SQLSTATE[HY000]: General error: 1 Cannot add a NOT NULL column with default value NULL in /var/www/nextcloud/3rdparty/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:88
```

I can provide more information if needed.

Amazing app by the way.

**Comment by marius-wieschollek** [comment]
I have published 2020.3.1-build3487 as nightly to the app store. It would be great if you can confirm if that resolves the issue for you.

**Comment by julien-nc** [comment]
Yes it does. The update ran successfully. Thanks for the lightning fix.
